paywall/src/unlock.js/setupPostOffices.ts

```diff
@@ -14,6 +14,29 @@ interface process {
 
 declare const process: process
 
+export function normalizeConfig(unlockConfig: any) {
+  if (
+    !unlockConfig ||
+    !unlockConfig.locks ||
+    typeof unlockConfig.locks !== 'object'
+  )
+    return unlockConfig
+  const lockAddresses = Object.keys(unlockConfig.locks)
+  if (!lockAddresses.length) {
+    return unlockConfig
+  }
+  const normalizedConfig = {
```

Comment:
this lower-cases all of the lock addresses as keys. This way configuration locks are always normalized to lowercase